test/Bridges/Constraint/vectorize.jl

```diff
@@ -1,82 +1,195 @@
+    )
     MOIT.linear6test(mock, config)
 
-    MOIU.set_mock_optimize!(mock,
-         (mock::MOIU.MockOptimizer) -> MOIU.mock_optimize!(mock, [0, 1/2, 1],
-             (MOI.VectorAffineFunction{Float64}, MOI.Nonpositives) => [[-1], [-2]],
-             (MOI.VectorAffineFunction{Float64}, MOI.Nonnegatives) => [[2], [0], [0]]),
-         (mock::MOIU.MockOptimizer) -> MOIU.mock_optimize!(mock, [1],
-             (MOI.VectorAffineFunction{Float64}, MOI.Nonpositives) => [[-1]],
-             (MOI.VectorAffineFunction{Float64}, MOI.Nonnegatives) => [[0]]))
+    MOIU.set_mock_optimize!(
+        mock,
+        (mock::MOIU.MockOptimizer) -> MOIU.mock_optimize!(
+            mock,
+            [0, 1/2, 1],
+            (MOI.VectorAffineFunction{Float64}, MOI.Nonpositives) => [[-1], [-2]],
+            (MOI.VectorAffineFunction{Float64}, MOI.Nonnegatives) => [[2], [0], [0]]),
+        (mock::MOIU.MockOptimizer) -> MOIU.mock_optimize!(
+            mock,
+            [1],
+            (MOI.VectorAffineFunction{Float64}, MOI.Nonpositives) => [[-1]],
+            (MOI.VectorAffineFunction{Float64}, MOI.Nonnegatives) => [[0]],
+        )
+    )
     # linear14 has double variable bounds for the z variable
     mock.eval_variable_constraint_dual = false
     MOIT.linear14test(bridged_mock, config)
     mock.eval_variable_constraint_dual = true
 
-    mock.optimize! = (mock::MOIU.MockOptimizer) -> MOIU.mock_optimize!(mock, ones(3),
-                           (MOI.VectorAffineFunction{Float64}, MOI.Zeros) => [[2]])
+    mock.optimize! = (mock::MOIU.MockOptimizer) -> MOIU.mock_optimize!(
+        mock, ones(3), (MOI.VectorAffineFunction{Float64}, MOI.Zeros) => [[2]]
+    )
     MOIT.psdt0vtest(bridged_mock, config)
 
-    ci = first(MOI.get(bridged_mock, MOI.ListOfConstraintIndices{MOI.ScalarAffineFunction{Float64}, MOI.EqualTo{Float64}}()))
+    ci = first(
+        MOI.get(
+            bridged_mock,
+            MOI.ListOfConstraintIndices{
+                MOI.ScalarAffineFunction{Float64}, MOI.EqualTo{Float64}
+            }()
+        )
+    )
 
     @testset "$attr" for attr in [MOI.ConstraintPrimalStart(), MOI.ConstraintDualStart()]
         @test MOI.supports(bridged_mock, attr, typeof(ci))
         MOI.set(bridged_mock, attr, ci, 2.0)
         @test MOI.get(bridged_mock, attr, ci) == 2.0
     end
 
-    test_delete_bridge(bridged_mock, ci, 3,
-                       ((MOI.VectorAffineFunction{Float64},
-                         MOI.Zeros, 0),))
+    test_delete_bridge(
+        bridged_mock,
+        ci,
+        3,
+        ((MOI.VectorAffineFunction{Float64}, MOI.Zeros, 0),)
+    )
+
+    @testset "_vectorized_convert" begin
+        @testset "SingleVariable" begin
+            model = MOI.Utilities.Model{Float64}()
+            x = MOI.add_variable(model)
+            f = MOI.SingleVariable(x)
+            f_vov = MOI.Bridges.Constraint._vectorized_convert(MOI.VectorOfVariables, f)
```

Comment:
If it's tested then we could maybe move it to `MOIU` and remove the first `_`

Reply:
I would prefer to call make function `Base.convert`. Then it can go in MOI proper. But you and Joaquim seem to dislike the `convert`...